docusaurus.config.js

```diff
@@ -69,7 +71,20 @@ module.exports = {
       },
     ],
   ],
-  scripts: [],
+  scripts: [
+    {
+      src: "https://widget.kapa.ai/kapa-widget.bundle.js",
+      "data-website-id": "1a0b2863-2767-4583-9d33-ded0095731e7",
+      "data-project-name": "Camunda",
+      "data-project-color": "#000000",
+      "data-project-logo":
+        "https://avatars.githubusercontent.com/u/2443838?s=200&v=4",
+      "data-modal-disclaimer":
+        "Welcome to Camunda 8 Smart docs, powered by AI. Accessing developer docs, forum posts and product blogs from the last year, responses are generated by combining various sources to formulate the best possible answer. If you have feedback please give a thumbs up or down as we continue to improve the AI.",
+      "data-modal-example-questions": `How do I get started with Camunda 8?,What is new in Camunda ${latestVersion}?`,
```

Comment:
This will prevent me from doing a panic-release after each new version, as we realize the sample question has become out of date.